pkg/prowgen/prowgen_images_discovery.go

```diff
@@ -153,13 +191,14 @@ func getPullStringsFromDockerfile(filename string) ([]string, error) {
 	scanner := bufio.NewScanner(file)
 	for scanner.Scan() {
 		line := scanner.Text()
-		if !strings.Contains(line, "FROM ") {
+		if !(strings.Contains(line, "FROM ") || strings.Contains(line, "--from=")) {
 			continue
 		}
 
 		match := registryRegex.FindString(line)
 		if match != "" {
 			images = append(images, match)
+			log.Println("Added match:", match)
```

Comment:
Only for debugging? Otherwise it might be good toadd a bit more context to the log line.

Reply:
Right. I will remove it.